plugin/group_replication/libmysqlgcs/src/bindings/xcom/xcom/xdr_utils.h

```diff
@@ -33,7 +33,6 @@ extern "C" {
     Initialize an array
  */
 #define def_init_xdr_array(name) \
-  MY_ATTRIBUTE((unused))         \
```

Comment:
No compilation warning regression?

Reply:
Yes, everything is clear. Removing `MY_ATTRIBUTE((unused)) ` was our quick and dirty way of suppressing modern gcc / clang warnings, then Oracle fixed this properly.
https://github.com/mysql/mysql-server/commit/1f610f7
We reverted these in 5.7, but never in 8.0